3_javascript/javascript2/week2/index.html

```diff
@@ -0,0 +1,56 @@
+<head>
+  <title>HyfBay</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
Super super nice that you put it into a proper page! 
Overall your homework this a very good job, but this extra effort is very good. Keep it up, it will pay off 🙏🏽

Reply:
> Super super nice that you put it into a proper page!
> Overall your homework this a very good job, but this extra effort is very good. Keep it up, it will pay off 🙏🏽

thanks @shpomp will do my best 🙏🏾